Packs/Infoblox/Integrations/Infoblox/test_data/TestDhcpLeaseLookupCommand/dhcp_lease_lookup_success_hr.md

```diff
@@ -0,0 +1,6 @@
+### DHCP lease lookup, found 3 records
```

Comment:
Missing 'Client Hostname' and 'Next Binding State' columns for some entries in human-readable output.